comfyui_custom_scripts/extensions/webuiRequests.js

```diff
@@ -1,170 +1,60 @@
-        return 'success';
-    }],
-]);
-
-async function longPolling(workflowTypeId, webuiClientKey, startupResponse) {
-    let clientResponse = startupResponse;
-
-    try {
-        while(true) {
-            const response = await api.fetchApi("/sd-webui-comfyui/webui_polling_server", {
-                method: "POST",
-                headers: {
-                    "Content-Type": "application/json",
-                },
-                cache: "no-store",
-                body: JSON.stringify({
-                    workflow_type_id: workflowTypeId,
-                    webui_client_id: webuiClientKey,
-                    response: clientResponse,
-                }),
-            });
-            const json = await response.json();
-            if (json.request !== '/sd-webui-comfyui/webui_request_timeout') {
-                console.log(`[sd-webui-comfyui] WEBUI REQUEST - ${workflowTypeId} - ${json.request}`);
-            }
-            clientResponse = await request_map.get(json.request)(json);
-        }
-    }
-    catch (e) {
-        console.error(e);
-        clientResponse = {error: e};
-    }
-    finally {
-        setTimeout(() => {
-            longPolling(workflowTypeId, webuiClientKey, clientResponse);
-        }, 100);
-    }
-}
-
 async function onElementDomIdRegistered(callback) {
     const iframeInfo = await iframeRegisteredEvent;
     console.log(`[sd-webui-comfyui][comfyui] REGISTERED WORKFLOW TYPE ID - "${iframeInfo.workflowTypeDisplayName}" (${iframeInfo.workflowTypeId}) / ${iframeInfo.webuiClientId}`);
 
-    event.source.postMessage(iframeInfo.workflowTypeId, event.origin);
     await patchUiEnv(iframeInfo.workflowTypeId);
     const clientResponse = 'register_cid';
-    console.log(`[sd-webui-comfyui][comfyui] INIT LONG POLLING SERVER - ${clientResponse}`);
+    console.log(`[sd-webui-comfyui][comfyui] INIT WS - ${clientResponse}`);
     await callback(iframeInfo.workflowTypeId, iframeInfo.webuiClientId, clientResponse);
 }
 
-async function patchUiEnv(workflowTypeId) {
-    await appReadyEvent;
-
-    if (workflowTypeId.endsWith('_txt2img') || workflowTypeId.endsWith('_img2img')) {
-        const menuToHide = document.querySelector('.comfy-menu');
-        menuToHide.style.display = 'none';
-        patchSavingMechanism();
-    }
-
-    await patchDefaultGraph(workflowTypeId);
-}
-
-function patchSavingMechanism() {
-    app.graph.original_serialize = app.graph.serialize;
-    app.graph.patched_serialize = () => JSON.parse(localStorage.getItem('workflow'));
-    app.graph.serialize = app.graph.patched_serialize;
-
-    app.original_graphToPrompt = app.graphToPrompt;
-    app.patched_graphToPrompt = () => {
-        app.graph.serialize = app.graph.original_serialize;
-        const result = app.original_graphToPrompt();
-        app.graph.serialize = app.graph.patched_serialize;
-        return result;
+onElementDomIdRegistered(async (workflowTypeId, webuiClientId, clientResponse) => {
```

Comment:
Do we really need to split these 2 functions? Maybe put the code here directly where we would call the callback above.

As a side note, if we call multiple times `onElementDomIdRegistered` right now, it will print multiple times in the console. Maybe the logging code should move to the event definition.